src/server/server.js

```diff
@@ -151,19 +151,24 @@ function isAuthenticated(req, res, next) {
 
 app.use("/public", express.static(path.join(__dirname, "public")));
 
+let redirectedUrl = "/"
```

Comment:
In this case it will work, but be very careful about putting global variables here because different concurrent requests to the paths will use (overwrite/read) this variable. Personally, I'd probably introduce a `const DEFAULT_REDIRECT_URL = '/'` in the top of the file and do any changes to it within the request functions (ref. ll 158 and 167) and fall back to the introduced `DEFAULT_REDIRECT_URL`.